beacon-chain/rpc/prysm/v1alpha1/validator/attester.go

```diff
@@ -49,13 +50,19 @@ func (vs *Server) ProposeAttestation(ctx context.Context, att *ethpb.Attestation
 		return nil, err
 	}
 
-	go func() {
-		attCopy := att.Copy()
-		if err := vs.AttPool.SaveUnaggregatedAttestation(attCopy); err != nil {
-			log.WithError(err).Error("Could not save unaggregated attestation")
-			return
+	if features.Get().EnableExperimentalAttestationPool {
```

Comment:
I don't think we should spin up a goroutine here as is the case with the old design. Adding an attestation is very fast.